pipelines/deploy.sh

```diff
@@ -41,38 +36,15 @@ echoDemarcation() {
 # TRIGGER DEPLOY
 # --------------------------------------------------------------------------------
 echoDemarcation "Deploy Cloud Formation Template ..."
+mapfile -t PARAMETER_OVERRIDES < <(jq -r '.[] | del(select(."ParameterKey" == "ServiceImageVersion")) | values | "\(.ParameterKey)=\(.ParameterValue)"' "$PARAM_FILE")
```

Comment:
We should ensure that jq is installed in the docker-container used in the pipeline

Reply:
Notice that the old getCfParam function already used the jq command.
By ensure, do you mean we should do a check in the beginning of the script and fail-fast if it is not installed?

Reply:
`jq` is available within the container. The script is used in the pipeline and works: https://github.com/linked-planet/ktorbase/actions/runs/945443561

Reply:
As Simon writes, we have automated test for GitHub actions but not yet for Bitbucket Pipelines. We are planning to do that as well.